bundles/org.openhab.binding.qbus/README.md

```diff
@@ -0,0 +1,110 @@
+# Qbus Binding
+
+![Qbus Logo](doc/Logo.JPG)
+
+This binding for Qbus communicates for all controllers of the Qbus home automation system.
+
+More information about Qbus can be found here:
+[Qbus](https://qbus.be)
+
+This is the link to the [Qbus forum](https://qbusforum.be). This forum is mainly in dutch and you can find a lot of information about the pre testings af this binding and offers a way to communicate with other users.
+
+We also host a site which contains a [manual](https://manualoh.schockaert.tk/) where you can find lot's of information.
+
+The controllers can not communicate directly with openHAB, therefore we developed a Client/Server application which you must install prior to enable this binding.
+More information can be found here:
+[Qbus Client/Server](https://github.com/QbusKoen/QbusClientServer)
+
+With this binding you can control and read almost every output from the Qbus system.
+
+## Supported Things
+
+The following things are supported by the Qbus Binding:
+
+- Dimmer 1 button, 2 button and clc as _dimmer_
+- Bistabiel, Timer1-3, Interval as _onOff_
+- Thermostats - normal and PID as _thermosats_
+- Scenes as _scene_
+- CO2 as _co2_
+- Rollershutter and rollerhutter with slats as _rollershutter_
+
+For now the folowing Qbus things are not yet supported but will come:
+
+- DMX
+- Timer 4 & 5
+- HVAC
+- Humidity
+- Renson
+- Duco
+- Kinetura
+- Energy monitor
+- Weather station
+
+
+## Discovery
+
+The discovery service is not yet implemented but the System Manager III software of Qbus generates things and item files from the programming, which you can use directly in openHAB.
+
+## Thing Configuration
+
+This is an example of the things configuration file:
+
+```
+Bridge qbus:bridge:CTD001122 [ addr="localhost", sn="001122", port=8447, refresh=10 ] {
+    dimmer                   1     "ToonzaalLED"      [ dimmerId=100 ]
+    onOff                    30    "Toonzaal230V"     [ bistabielId=76 ]
+    thermostat               50    "Service"          [ thermostatId=99 ]
+    scene                    70    "Disco"            [ sceneId=36 ]
+    co2                      100   "Productie"        [ co2Id=26 ]
+    rollershutter            120   "Roller1"          [ rolId=268 ]
+    rollershutter_slats      121   "Roller2"          [ rolId=264 ]
+}
+```
+The Bridge connects to the QbusServer, so if the Client/Server application is installed on the same machine then localhost can be used as address. sn is the serial nr of your controller, port should allways be 8447, except in special installations as it is the communication port of the Server application. refresh is a time in minutes which will check the status of the server and reconnects if connection is broken after this time.
```

Comment:
Can you replace the configuration parameter text by a table? See here for example https://www.openhab.org/addons/bindings/telegram/